llvm/tools/llvm-cas-test/llvm-cas-test.cpp

```diff
@@ -0,0 +1,252 @@
+//===- llvm-cas-test.cpp - CAS stress tester ------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include "llvm/ADT/BitmaskEnum.h"
+#include "llvm/CAS/ActionCache.h"
+#include "llvm/CAS/BuiltinUnifiedCASDatabases.h"
+#include "llvm/CAS/ObjectStore.h"
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Path.h"
+#include "llvm/Support/Program.h"
+#include "llvm/Support/RandomNumberGenerator.h"
+#include "llvm/Support/ThreadPool.h"
+
+using namespace llvm;
+using namespace llvm::cas;
+
+static cl::opt<std::string>
+    CASPath("cas-path", cl::desc("CAS path on disk for testing"), cl::Required);
```

Comment:
Should this be `-cas` to be consistent with `llvm-cas`? I admit I find it hard to remember which tools use each spelling.